frontend/styles/var/padding.scss

```diff
@@ -20,3 +20,4 @@ $gap-form: $pad-large;
 $gap-form-component: $pad-small;
 $gap-data-sets: $pad-medium;
 $gap-table-elements: $pad-medium;
+$gap-modal-component: $pad-large;
```

Comment:
I don't intuitively understand what this is for based on the name. Is there a more descriptive name
we can use?